corehq/ex-submodules/casexml/apps/phone/tests/test_new_sync.py

```diff
@@ -167,3 +169,60 @@ def test_new_then_old_sync(self):
                                        params=RestoreParams(sync_log_id=sync_log._id))
         with self.assertRaises(IncompatibleSyncLogType):
             restore_config.get_payload()
+
+
+@override_settings(TESTS_SHOULD_USE_CLEAN_RESTORE=True)
+class TestNewSyncSpecifics(TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        cls.domain = uuid.uuid4().hex
+        cls.project = Domain(name=cls.domain)
+        cls.user_id = uuid.uuid4().hex
+        cls.user = User(user_id=cls.user_id, username=uuid.uuid4().hex,
+                        password="changeme", date_joined=datetime(2014, 6, 6))
+
+    def test_legacy_support_toggle(self):
+        restore_config = RestoreConfig(self.project, user=self.user)
+        factory = CaseFactory(domain=self.project.name, case_defaults={'owner_id': self.user_id})
+        # create a parent and child case (with index) from one user
+        parent_id, child_id = [uuid.uuid4().hex for i in range(2)]
+        parent_id, child_id = ['parent', 'child']
```

Comment:
whoops that was just for testing locally https://github.com/dimagi/commcare-hq/pull/8174/